src/ui/public/agg_response/hierarchical/build_hierarchical_data.js

```diff
@@ -51,7 +50,7 @@ export default function buildHierarchicalDataProvider(Private, Notifier) {
     }
 
     const firstAgg = buckets[0];
-    const aggData = resp.aggregations[firstAgg.id];
+    const aggData = resp.aggregations ? resp.aggregations[firstAgg.id] : null;
```

Comment:
if resp.aggregations is null here, does it make sense to continue with the rest of the function, or can you bail out early?